Revamp the Overview article (#160)
AdguardTeam/KnowledgeBaseDNS: docs/private-dns/overview.md

![Private AdGuard DNS dashboard main](https://cdn.adtidy.org/public/Adguard/Blog/private_adguard_dns/main.png)

## General

<iframe width="560" height="315" class="youtube-video" src="https://www.youtube-nocookie.com/embed/ME3_Ms9LO8M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

> Private AdGuard DNS offers all the advantages of a public AdGuard DNS server, including traffic encryption and domain blocklists. Moreover, it provides additional features such as flexible customization, DNS statistics, and parental control. All these capabilities are easily accessible and managed via a user-friendly dashboard.

Reviewer: ```suggestion
Private AdGuard DNS offers all the advantages of a public AdGuard DNS server, including traffic encryption and domain blocklists. It also offers additional features such as flexible customization, DNS statistics, and Parental control. All these options are easily accessible and managed via a user-friendly dashboard.
```

### Why you need private AdGuard DNS

Today, you can connect anything to the Internet: TVs, refrigerators, smart bulbs, or speakers. But along with the undeniable conveniences you get trackers and ads. A simple browser-based ad blocker will not protect you in this case, but AdGuard DNS, which you can set up to filter traffic, block content and trackers, has a system-wide effect.

At one time, the AdGuard product line included only [public AdGuard DNS](../public-dns/overview.md) and [AdGuard Home](https://github.com/AdguardTeam/AdGuardHome). These solutions work fine for some users, but for others, the public AdGuard DNS lacks the flexibility of configuration, while the AdGuard Home lacks simplicity. That's where private AdGuard DNS comes into play. It has the best of both worlds: it offers customizability, control and information - all through a simple easy-to-use dashboard.

> ### The difference between private and public AdGuard DNS

Reviewer: ```suggestion
### The difference between public and private AdGuard DNS
```

> Here is a simple comparison of features available in public AdGuard DNS and private AdGuard DNS servers.

Reviewer: ```suggestion
Here is a simple comparison of features available in public and private AdGuard DNS.
```

Every device that you add in the AdGuard DNS panel has its own unique address that can be used if the device supports modern encrypted DNS protocols (DoH, DoT, and DoQ).

> ### For devices that do not support DoH, DoT, and DoQ

Reviewer 2: Linking IP is not the only way, there's also dedicated IPv6 addresses which is a better option (but not everyone can use it).

### For devices that do not support DoH, DoT, and DoQ

If the device does not support encrypted DNS and you have to use plain DNS, there are two more ways to allow AdGuard DNS to recognize the device — use dedicated IP addresses or link device's IP address.

:::note

> Use plain DNS addresses only if you have no other options: this reduces the security of DNS requests. If you decide to use plain DNS, we recommend choosing dedicated IP addresses.

Reviewer: ```suggestion
Use plain DNS addresses only if you have no other options: this reduces the security of DNS requests. If you decide to use plain DNS, we recommend that you choose dedicated IP addresses.
```

:::

#### Dedicated IP addresses

For every device that you connect to AdGuard DNS, you'll be offered two dedicated IPv6 addresses that you can enter in your device settings. Using both IPv6 addresses is not mandatory, but often devices might request you to enter two IPv6 addresses.

When you connect to them, AdGuard DNS will be able to determine which particular device is sending DNS requests and display statistics for it. And you'll be able to configure DNS rules specifically for this device.

> Unfortunately, not all service providers offer IPv6 support, and not all devices allow you to configure IPv6 addresses. If it is your case, you might need to rely on the Linked IP method.

Reviewer: ```suggestion
Unfortunately, not all service providers offer IPv6 support, and not all devices allow you to configure IPv6 addresses. If this is your case, you may have to rely on the Linked IP method.
```

#### Linked IP

> If you connect your device to AdGuard DNS through Linked IP the service counts all plain DNS requests that come from that IP address towards that "device". With this connection method, you would have to reconnect manually or via a special program every time the device's IP changed, which occurred after every reboot.

Reviewer: ```suggestion
If you connect your device to AdGuard DNS via Linked IP, the service will count all plain DNS requests coming from that IP address towards that "device". With this connection method, you would have to reconnect manually or through a special program each time the device's IP changes, which happens after each reboot.
```

![Private AdGuard DNS dashboard query log](https://cdn.adtidy.org/public/Adguard/Blog/private_adguard_dns/query_log.png)

## Server settings

This section features a range of settings allowing you to customize the operation of private AdGuard DNS, ensuring the Internet functions exactly as you desire.

### Blocklists management

> With *Blocklists* feature you can set which domains you want to block and which you don't. Сhoose from wide variety of blocklists for different purposes.

Reviewer: ```suggestion
The *Blocklists* feature allows you to specify which domains you want to block and which you don't. Choose from a variety of blocklists for different purposes.
```

![Private AdGuard DNS dashboard blocklists](https://cdn.adtidy.org/public/Adguard/Blog/private_adguard_dns/blocklists.png)

### Security settings

> Even if you are aware of all the tricks of online scammers, there's always a risk of accidentally clicking a harmful link. To protect yourself from such accidents, go to the *Security settings* section and check the boxes next to the options listed there.

Reviewer: ```suggestion
Even if you're aware of all the tricks online scammers use, there's always a chance you'll accidentally click a malicious link. To protect yourself from such accidents, go to the *Security settings* section and check the boxes next to the options listed there.
```

> Even if you're aware of all the tricks online scammers use, there's always a chance you'll accidentally click a malicious link. To protect yourself from such accidents, go to the *Security settings* section and check the boxes next to the options listed there.

Reviewer 2: Screenshots?

Author: i'll add them later along with updating other screenshots of this article after v2.5 release -- if you don't mind

The *Block malicious, phishing, and scam domains* feature will block domains found in the dedicated database. And the *Block newly registered domains* will block all domains registered less than 30 days ago, which are often considered risky for your online privacy.

### User rules

> For times when pre-installed *Blocklists* with thousands of rules are not enough, we have a handy function called *User rules*. Here you can add custom rules manually to block/unblock a certain domain or import custom rules lists (check out [DNS filtering rules syntax](../general/dns-filtering-syntax.md)). You can export the lists.

Reviewer: ```suggestion
For cases where pre-installed blocklists with thousands of rules are not enough, we have a handy feature called *User rules*. Here you can manually add custom rules to block/unblock a specific domain or import custom rule lists (see [DNS filtering rules syntax](../general/dns-filtering-syntax.md)). You can export the lists.
```
